pkg/nsx/services/builder.go

```diff
@@ -839,12 +845,13 @@ func (service *SecurityPolicyService) mergeSelectorMatchExpression(matchExpressi
 // Given NSX currently doesn't support 'In' LabelSelector, to keep design simple,
 // only allow just one 'In' LabelSelector in matchExpressions with at most of five values in it.
 func (service *SecurityPolicyService) validateSelectorOpIn(matchExpressions []v1.LabelSelectorRequirement,
-	matchLabels map[string]string) (int, error) {
-	var mexprInOpCount = 0
-	var mexprInValueCount = 0
+	matchLabels map[string]string,
+) (int, error) {
+	mexprInOpCount := 0
+	mexprInValueCount := 0
 	var err error = nil
-	var errorMsg = ""
-	var exists = false
+	errorMsg := ""
```

Comment:
Why we changed all `var errorMsg = ""` to `errorMsg := ""`?

Reply:
This sort of changes were done by golangci-lint tool in vscode. I think the tool prefers this code style way.